java/Anton-Kozyk/src/com/zoolatech/lecture3/tasks/_3/Triangle.java

```diff
@@ -0,0 +1,63 @@
+package com.zoolatech.lecture3.tasks._3;
+
+import java.util.Objects;
+
+public non-sealed class Triangle extends Figure implements Shape {
+    private double sideA;
+    private double sideB;
+    private double sideC;
+
+    Triangle(double sideA, double sideB, double sideC){
+        this.sideA = sideA;
+        this.sideB = sideB;
+        this.sideC = sideC;
+    }
+
+    @Override
+    public double findPerimeter() {
+        double trianglePerimeter = sideA + sideB + sideC;
```

Comment:
since it's clear from the method name what will be the result, you can return the value directly here, without introducing of a local variable